### ISSUE TITLE
Interspaced Rtable1-zebra？

### ISSUE THREAD

**Issue by SmartChen96** [benjaminrich/table1#114]
Hi Benjaminrich,

Thanks for the powerful package for creating descriptive statistics.  I am wondering if it is possible to let background filling be alternated for every other row when setting `topclass="Rtable1-zebra"`. Due to the need to show only one-row continuous variable, the default format of shade rows for "Rtable1-zebra" seems incorrect.  Thus, here comes this question.

<img width="586" alt="image" src="https://github.com/benjaminrich/table1/assets/89902816/84645ae8-d81d-4536-be91-97f32815f76c">

**Comment by benjaminrich** [comment]
I'm not sure how you generated this, but I can't reproduce it. When I do it, it looks fine:

![image](https://github.com/benjaminrich/table1/assets/17572252/3e77f4ad-3720-4148-92ce-dfa2ca440c74)

I used this code:
```r
rndr <- function(x, name, ...) {
    stats <- c(.="Mean (SD)")
    if (name=="age") {
        stats <- c(.="Median [Min, Max]")
    }

    render.default(x, render.continuous=stats, ...)
}

table1(~ age + sex + wt | treat, data=dat, topclass="Rtable1-zebra", render=rndr)
```

**Comment by SmartChen96** [comment]
Sorry to bother you with such a question. I've also just started using this function, and the issue seems to be resolved for now, although I'm not sure why it worked fine with a different dataset. 

Additionally, I'd like to ask for your help with another issue: I've browsed through old posts and tried to set the decimal places for numbers and percentages in the table, but I'm not sure how to do that within the custom function 1：
`rndr <- function(x, name, ...) {
  cont <- ifelse(name %in% vars.normal, "Mean (SD)", "Median (Q1 - Q3)")
  y <- render.default(x, name, render.continuous=cont, ...)
  if (is.logical(x)) {y[2]} 
  else if (is.factor(x) & length(levels(x))==2) {
    y[names(y) != levels(x)[1]] # Exclude the first (reference) level
  } else {y}
}`
 I found that it can be done in the `render.continuous `function, but it seems that the two approaches conflict with each other. Of course, I'm also not clear on how to efficiently set the decimal places for both numbers and percentages at the same time. I also wonder if there is a more efficient way to set decimal places for numbers and percentages.  A custom function like function 1 allows for many special settings and changes, for me, is highly recommended.
